src/lib/src/common/markers/workspace.marker.update.ts

```diff
@@ -0,0 +1,4 @@
+export class WorkspaceMarkerUpdate {
+  path: string;
+  markers: any;
```

Comment:
a comment would be helpful to indicate what is (usually) expected in this field ('js object holding markers in fields e.g. { errors: 0, errmsg: 'some error' }')